docs/data/stashcache/install-cache.md

```diff
@@ -5,100 +5,130 @@ This document describes how to install a StashCache cache service.  This service
+```yaml
+  MY_AUTH_STASHCACHE_CACHE:
+    FQDN: my-auth-cache.example.net
+    Service: XRootD cache server
+      Description: StashCache cache server
+    AllowedVOs:
+      - OSG
+    DN: /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=my-auth-cache.example.net
+```
+
+
+### Finalizing registration
+
+Once initial registration is complete, you may start the installation process.
+In the meantime, open a [help ticket](https://support.opensciencegrid.org) with your cache name.
+Mention in your ticket that you would like to "Finalize the cache registration."
+
+
+Installing the Cache
+--------------------
+
+The StashCache software consists of an XRootD server with special configuration and supporting services.
+To simplify installation, OSG provides convenience RPMs that install all required
+packages with a single command:
+
+    :::console
+    root@host # yum install --enablerepo=osg-testing stash-cache
 
 
 Configuring the Cache
 ---------------------
 
 First, you must create a "cache directory", which will be used to store downloaded files.
-By default this is `/stash`.
+By default this is `/mnt/stash`.
 We recommend using a separate file system for the cache directory,
 with at least 1 TB of storage available.
 
 !!! note
     The cache directory must be writable by the `xrootd:xrootd` user and group.
 
-The `stashcache-cache-server` provides a default configuration file,
-`/etc/xrootd/xrootd-stashcache-cache-server.cfg`, which must be customized for your cache.
-
-The most common lines to customize are:
+The `stash-cache` package provides default configuration files in `/etc/xrootd/xrootd-stash-cache.cfg` and `/etc/xrootd/config.d/`.
+Administrators may provide additional configuration by placing files in `/etc/xrootd/config.d/1*.cfg` (for files that need to be processed BEFORE the OSG configuration) or `/etc/xrootd/config.d/9*.cfg` (for files that need to be processed AFTER the OSG configuration).
 
-* `all.sitename YOUR_SITE_NAME`: The registered OSG resource name.  This **must** be changed.
-* `set cachedir = /stash`: The location of the cache directory for this service.
-* `*.trace`, `pss.setopt`: These control the logging verbosity of the cache.  The defaults are relatively high
-  in order to aid in debugging.  These lines can be commented out to reduce logging; however, if issues occur,
-  OSG support may ask you to re-enable them.
+You _must_ configure every variable in `/etc/xrootd/10-common-site-local.cfg`.
 
-The Authfile specifies which files and directories can be read,
-relative to the cache directory (`cachedir` in the main config).
+The mandatory variables to configure are:
 
-An example:
-
-```console
-root@host # cat /etc/xrootd/Authfile-noauth
-u * /user/ligo -rl / rl
-```
+- `set rootdir = /mnt/stash`: the mounted filesystem path to export.  This document refers to this as `/mnt/stash`.
+- `set sitename = YOUR_SITE_NAME`: the resource name registered with the OSG.
```

Comment:
```
site name = resource name
```

Have we not learned from the sins of the Gratia configuration?!

Reply:
Good call. Good thing I haven't tagged xcache 1.0.2 yet.